examples/nucleo_f429zi/pat9125el/project.xml

```diff
@@ -0,0 +1,13 @@
+<library>
+  <extends>modm:board:nucleo-f429zi</extends>
+  <options>
+    <option name="modm:build:build.path">../../../build/nucleo_f429zi/blink</option>
```

Comment:
This is the reason the CI is failing, this path needs to be unique, since we’re compiling in parallel and this overwrites the blinky project.